src/tsickle.ts

```diff
@@ -75,6 +75,19 @@ function isValidClosurePropertyName(name: string): boolean {
   return /^[a-zA-Z_][a-zA-Z0-9_]*$/.test(name);
 }
 
+/**
+ * TypeScript allows parameters named "arguments", but Closure
+ * disallows this, even in externs.
+ */
+function makeLegalExternsParam(name: string): string {
+  switch (name) {
+    case 'arguments':
+      return 'tsickle_arguments';
```

Comment:
Change to `arguments_tsickle` so that the original intended name is more clearly readable?